flink-runtime/src/main/java/org/apache/flink/runtime/state/heap/HeapRestoreOperation.java

```diff
@@ -201,11 +201,13 @@ private void readStateHandleStateData(
             long offset = groupOffset.f1;
 
             // Check that restored key groups all belong to the backend.
```

Comment:
I wonder whether this comment should still be kept.

Reply:
Good point, I removed the comment.